toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -428,10 +428,17 @@ chroot_and_install_rpms unzip
 build_rpm_in_chroot_no_install gperf
 chroot_and_install_rpms gperf
 
+# Python3 and pam need libxcrypt
+build_rpm_in_chroot_no_install libxcrypt
+chroot_and_install_rpms libxcrypt
+
 # Python3 needs to be installed for RPM to build
 build_rpm_in_chroot_no_install python3
 chroot_and_install_rpms python3 python3
 
+build_rpm_in_chroot_no_install python-setuptools
+chroot_and_install_rpms python-setuptools python3-setuptools
```

Comment:
I'm a bit confused by `python-setuptools`. I see the `python-setuptools.spec` build the `python3-setuptools` package but I think it doesn't build `python-setuptools`. Nor does it `Provides: python-setuptools`. Where is `python-setuptools` coming from?

Reply:
`python-setuptools` is the name of the spec - required for the first parameter. `python3-setuptools` is the name of the rpm(s) to install

Reply:
Ah, got it, thanks!:)